src/twisted/web/test/test_webclient.py

```diff
@@ -348,7 +348,11 @@ def tearDown(self):
             proto.transport.loseConnection()
         if connections:
             msg("Some left-over connections; this test is probably buggy.")
-        return self.port.stopListening()
+        self.port.stopListening()
```

Comment:
I pass self.port to the following function, which should achieve the same effect?